js/src/product-feed/product-statistics/index.scss

```diff
@@ -35,25 +35,40 @@
 			}
 		}
 
+		.components-flex {
+			align-items: flex-start;
+		}
+
 		.components-flex + .components-flex {
 			margin-top: $grid-unit-10;
 		}
 
 		.components-flex__item {
 			color: $gray-900;
+			padding-top: 0.25 * $grid-unit;
 
 			&:first-child {
 				min-width: 130px;
 			}
 
-			&:nth-child(2) {
-				display: inline-flex;
-				align-items: center;
-				fill: currentColor;
-				color: #008a20;
+			&.gla-status-icon {
+				min-width: 30px;
+				padding-top: 0;
 			}
 
-			&:last-child {
+			.gla-status-label {
+				color: $black;
```

Comment:
❔ I didn't find a case that uses the `$back` color, and it looks like the incorrect text color.

Reply:
Not sure what do you mean with this. Pending review as well as Under review and Warning uses Black text in the label.

Reply:
adjusted the color to be more accurate with design here: 
https://github.com/woocommerce/google-listings-and-ads/pull/1425/commits/de2c2c26c79778346945648efa656110faa97d8b

Reply:
Thanks for the adjustment! The same `font-color` is inherited from the above `.components-flex__item`, so it could be omitted as well.